backend-rust/src/graphql_api/account.rs

```diff
@@ -749,12 +749,7 @@ impl Account {
                 ON
                     blocks.height = account_statements.block_height
                 WHERE
-                    entry_type IN (
-                        'FinalizationReward',
-                        'FoundationReward',
-                        'BakerReward',
-                        'TransactionFeeReward'
-                    )
+                    entry_type = ANY($6)
```

Comment:
I am not sure these changes are improvements, since the postgresql planner now have less information to work with.
In general, it is better to have constants as part of the query.